lib/galaxy/webapps/galaxy/services/datasets.py

```diff
@@ -98,6 +98,15 @@ class RequestDataType(str, Enum):
     in_use_state = "in_use_state"
 
 
+class ConcreteObjectStoreQuotaSourceDetails(Model):
+    source: Optional[str] = Field(
+        description="The quota source label corresponding to the object store the dataset is stored in (or would be stored in)"
+    )
+    enabled: bool = Field(
```

Comment:
Does the description match the `enabled` variable ? Given the description should this maybe be `tracks_quota` ?